src/neptune/metadata_containers/metadata_containers_table.py

```diff
@@ -55,6 +56,9 @@ def __init__(
         self._id = _id
         self._attributes = attributes
 
+    def __repr__(self) -> str:
+        return f"TableEntry({self._container_type}, id={self._id})"
```

Comment:
Let's remove it as I see that we have to rename a couple of properties here to be more up-to-date.